src/lazy/expanded/compiler.rs

```diff
@@ -408,7 +420,7 @@ impl TemplateCompiler {
 
         // TODO: Enforce 'identifier' syntax subset of symbol
         let template_name =
-            Self::expect_nullable_name("a macro name", &mut values)?.map(|name| name.to_owned());
+            Self::expect_nullable_name("a macro name", &mut values)?.map(|name| name.into());
```

Comment:
🗺️ Macro names are now `Rc<str>` instead of `String`. This allows them to be cheaply cloned and shrinks the stack size of the name from 24 bytes (`String`) to 16 bytes (`Rc<str>`).